routes/soldiers.js

```diff
@@ -0,0 +1,28 @@
+const url = require("url");
+const serverUrl = require("../config").serverUrl;
+const utils = require("../utility/utility");
+
+function handleRequest(method, url, data, router) {
```

Comment:
how about requiring the single router from the router file as it is a non changing constant and not a parameter